README.md

```diff
@@ -177,3 +177,14 @@ end note
 ```
 
 For further guides and syntax, check out the official plantuml guide: https://plantuml.com/guide
+
+
+### Terraform deployment and destroy
+
+```
+terraform plan -var "client_id=<ID>" -var "client_secret=<SECRET>" -var "secret_key=<SECRET_KEY" -out terraform.tfplan
+```
+
+```
+terraform destroy -var "client_id=<ID>" -var "client_secret=<SECRET>" -var "secret_key=<SECRET_KEY"
+```
```

Comment:
A little bit of context for these commands would be good - it might sound stupid but my brain would default to assuming the second is the `apply` command once I've seen the first is `plan` so I can easily imagine accidentally running that!